### PR TITLE
Update Ably queues diagram

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Update the Ably Queues overview diagram.

* Epic [EDU-719](https://ably.atlassian.net/browse/EDU-719)
* [EDU-929](https://ably.atlassian.net/browse/EDU-929)

## Review

View the updated diagram: 

* [Ably Queues](https://ably-docs-edu-929-updat-0oaksm.herokuapp.com/general/queues/)

### CODE REVIEW

**Review by m-hulbert — COMMENTED** [review]
Minor point... but do you think we should use something other than a van sending lat/long in the example. This has some overlap and ability to cause confusion now that we offer AAT I'd have thought?

**Review by marklewin — COMMENTED** [review]
I like this, but would suggest putting the AAT stuff in a separate "note"-style callout. A quick read of this as-is might make it look like you _need_ AAT to do this? What we should perhaps be saying is that this is how you would build such a solution from scratch, but if you want all the mapping, etc out of the box, consider AAT.

**Comment by tbedford** [comment]
Yes, I don't want to muddy the waters, and you're right, it may imply it's a requirement (it's not). I will have a rethink.

File: content/general/queues.textile [hunk 1]
```diff
@@ -46,6 +46,8 @@ Ably combines both pub/sub and queueing functionality in its "platform":https://
   <img src="/images/diagrams/ably-queues-overview.png" style="width: 100%" alt="Ably Queues overview">
 </a>
 
+**Note:** It's not a requirement for this scenario, but Ably has the "Asset Tracking SDK":/asset-tracking, which can provide a complete and convenient asset tracking solution. You can read more about what's provided in the "docs":/asset-tracking.
```
Comment (m-hulbert) [inline content/general/queues.textile:49]: Not sure you need the second sentence here?

Comment (tbedford) [reply content/general/queues.textile]: I've changed it a bit. I decided it was worth guiding to both the product page and the docs. The product page also has a nice big box linking to the docs too.

**Review by marklewin — APPROVED** [review]
LGTM

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**